src/main/resources/grammar.txt

```diff
@@ -0,0 +1,5 @@
+S aAd
```

Comment:
I think it's best to consider the grammar being an array of space-separated strings, instead of of a single string.

This would make a rule like the one below possible (which is not the case in the current implementation):

expr term ADD term

where 'expr' and 'term' are labels of non-terminals, and ADD is the label of a token.

Can you fix that? Thanks
```suggestion
S a A d
```

Reply:
That make senses, I gonna fix it!